ansible_base/authentication/authenticator_plugins/saml.py

```diff
@@ -206,10 +206,10 @@ def validate(self, attrs):
             security_settings = set(attrs.get('SECURITY_CONFIG').keys())
             invalid_security_settings = security_settings.difference(valid_security_settings)
         except Exception as e:
-            raise ValidationError(f"Failed to load config: {e}")
+            raise ValidationError(_("Failed to load config: %(e)s") % {"e": e})
 
         if invalid_security_settings:
-            raise ValidationError({'SECURITY_CONFIG': f"Invalid keys: {', '.join(invalid_security_settings)}"})
+            raise ValidationError(_({'SECURITY_CONFIG': f"Invalid keys: {', '.join(invalid_security_settings)}"}))
```

Comment:
@john-westcott-iv @relrod do we need to change any formatting here? this is a new one to me.

Reply:
If we were going to do anything here I'd probably just `_()` on the "Invalid keys" string.